src/types/RootState.ts

```diff
@@ -9,24 +9,16 @@ import { TransactionState } from 'app/state/transaction/types'
 import { LedgerState } from 'app/state/ledger/types'
 import { StakingState } from 'app/state/staking/types'
 import { FatalErrorState } from 'app/state/fatalerror/types'
-// [IMPORT NEW CONTAINERSTATE ABOVE] < Needed for generating containers seamlessly
```

Comment:
Let's keep this https://github.com/oasisprotocol/oasis-wallet-web/blob/5cebb40d55c0a51badaf23f46a9303781453e5a3/internals/generators/slice/index.ts#L76